site/docs/v1/tech/apis/_key-search-request-parameters.adoc

```diff
@@ -0,0 +1,60 @@
+
+// parameter_prefix is either blank for parameters or "search." for body
+
+[.api]
+
+[field]#{parameter_prefix}algorithm# [type]#[String]# [optional]#Optional#::
+The algorithm used to generate the key.
++
+The possible values are:
++
+ * `ES256` - ECDSA using P-256 curve and SHA-256 hash algorithm
+ * `ES384` - ECDSA using P-384 curve and SHA-384 hash algorithm
+ * `ES512` - ECDSA using P-521 curve and SHA-512 hash algorithm
+ * `RS256` - RSA using SHA-256 hash algorithm
+ * `RS384` - RSA using SHA-384 hash algorithm
+ * `RS512` - RSA using SHA-512 hash algorithm
+ * `HS256` - HMAC using SHA-256 hash algorithm
+ * `HS384` - HMAC using SHA-384 hash algorithm
+ * `HS512` - HMAC using SHA-512 hash algorithm
+
+[field]#{parameter_prefix}name# [type]#[String]# [optional]#Optional#::
+The case-insensitive string to search for in the Key name. This can contain wildcards using the asterisk character (`*`). If no wildcards are present, the search criteria will be interpreted as `pass:[*value*]`. If this field is omitted, no filtering will be applied to the `name` field.
+
+[field]#{parameter_prefix}numberOfResults# [type]#[Integer]# [optional]#Optional# [default]#defaults to `25`#::
+The number of results to return from the search.
+
+[field]#{parameter_prefix}orderBy# [type]#[String]# [optional]#Optional# [default]#defaults to `name ASC`#::
+The field to order the search results as well as an order direction.
++
+The possible values are:
++
+ * `algorithm` - the algorithm used to generate the Key. `null` values are treated as smaller than all populated values.
+ * `expiration` - the expiration link:/docs/v1/tech/reference/data-types#instants[instant] of the Key. `null` values are treated as smaller than all populated values.
+ * `id` - the UUID of the Key
+ * `insertInstant` - the link:/docs/v1/tech/reference/data-types#instants[instant] when the Key was created
+ * `name` - the Key name
+ * `type` - the Key type
++
+The order direction is optional. Possible values of the order direction are `ASC` or `DESC`. If omitted, the default sort order is `ASC`.
++
+For example, to order the results by the insert instant in a descending order, use `insertInstant DESC`.
+
+[field]#{parameter_prefix}startRow# [type]#[Integer]# [optional]#Optional# [default]#defaults to `0`#::
+The offset into the total results. In order to paginate the results, increment this value by the [field]#numberOfResults# for subsequent requests.
+
+[field]#{parameter_prefix}type# [type]#[String]# [optional]#Optional#::
+The key type. The possible values are:
++
+ * `EC`
+ * `RSA`
+ * `HMAC`
++
+
```

Comment:
```suggestion
```

Do you need this extra space?

Reply:
Yes
![image](https://user-images.githubusercontent.com/3409780/217323617-ae577aeb-fa81-4b28-93a4-70a91c217aab.png)